.gitmodules

```diff
@@ -9,7 +9,7 @@
 [submodule "clang"]
 	path = clang
 	url = https://github.com/RadeonOpenCompute/hcc-clang-upgrade.git
-	branch = clang_tot_upgrade
+	branch = feature_refactor_dfcs
```

Comment:
This should be changed back to clang_tot_upgrade

Reply:
Eventually yes, but for now it allows people to just clone and use it directly, without having to check something out in the submodule.

Reply:
Ah, I didn't notice you merged the Clang PR, thanks. However, there'll be a follow-up to handle the hc-function-calls not-as-linker-flag scenario that you outlined above.

Reply:
I don't believe the branch field will make a difference.  The gitref contains the commit id, and the path and url fields give git all it needs to get the submodule.